templates/components/form/basic_inputs_macros.html.twig

```diff
@@ -208,6 +208,7 @@
 {% macro date(name, value, options = {}) %}
     {% set options = {
         'rand': random(),
+        'noCalendar': false,
```

Comment:
`noCalendar` is really specific to `flatpickr`, `enableDate` would probably more comprehensive.
```suggestion
        'enableDate': true,
```